bundle/manifests/gpu-operator-certified.clusterserviceversion.yaml

```diff
@@ -209,13 +209,11 @@ spec:
     - name: container-toolkit-image
       image: nvcr.io/nvidia/k8s/container-toolkit@sha256:d934033f9ff89087d555850eb1da150eaaa27b11e5ad21db1b874b8bc0b8d35b
     - name: driver-image
+      image: nvcr.io/nvidia/driver@sha256:550e44fc7e400e9700e6f85201408dad5f40270b6635787f8fd983350b976963
+    - name: driver-image-550
       image: nvcr.io/nvidia/driver@sha256:6fe74322562c726c8fade184d8c45ebae3da7b1ea0a21f0ff9dc42c66c65e692
     - name: driver-image-535
       image: nvcr.io/nvidia/driver@sha256:026f8f2d29b7058ecaaa5b98666e28e1b3646b73e7d7cda3dc8026d47a929152
-    - name: driver-image-560
```

Comment:
R560 and R565 have been EOL'd for awhile now, so I went ahead and just removed them.

NOTE: I am keeping R550, as the next release will be the last one bundling a R550 driver (even though R550 will have been EOL'd at the time of the release)

Reply:
Sounds good. Can you also update the image digest in the example NVIDIADriver CR?